integreat_compass/cms/templates/overlay/offer-detail-overlay.html

```diff
@@ -82,6 +82,81 @@ <h3 class="text-xl mt-4 font-semibold">{% translate "More details" %}</h3>
                         </div>
                     </div>
                 </div>
+                <div class="md:flex justify-left gap-12">
+                    {% if perms.cms.add_comment %}
+                        <div class="shrink-0 xl:mr-0 xl:w-1/3 text-gray-500">
+                            <form class="comment-form"
+                                  method="post"
+                                  action="{% url 'cms:protected:add_comment' %}">
+                                {% csrf_token %}
+                                <label for="id_offer-comment-{{ offer.pk }}">{% translate "Comment on this offer" %}</label>
+                                <p class="hidden error-message mt-2 bg-red-100 border-l-4 border-red-500 text-red-500 px-4 py-3 mb-2">
+                                    {% translate "An error occurred. Please try again." %}
+                                </p>
+                                <textarea id="id_offer-comment-{{ offer.pk }}"
+                                          name="comment"
+                                          rows="8"
+                                          cols="50"
+                                          maxlength="{{ max_comment_length }}"
+                                          required></textarea>
+                                <input type="hidden"
+                                       id="id_offer-id-{{ offer.pk }}"
+                                       name="offer_id"
+                                       value="{{ offer.pk }}">
+                                <button type="submit" class="btn mt-2">{% translate "Add comment" %}</button>
+                                <div class="float-right">
+                                    <span class="comment-length">0</span>/{{ max_comment_length }}
+                                </div>
+                            </form>
+                        </div>
+                    {% endif %}
+                    <div class="shrink grow-0 {% if not offer.comments %}hidden{% endif %}">
+                        <div class="mb-4">
```

Comment:
```suggestion
                        <div class="my-4">
```
This is quite picky, but I would personally prefer for the two columns to start on the same height :sweat_smile: but feel free to disagree :blush: